cmake/CMakeLists.txt

```diff
@@ -1107,6 +1107,9 @@ include_directories(
   ${REPO_ROOT}/include/onnxruntime/core/session
 )
 
+# DLPack is a header-only dependency
+set(DLPACK_INCLUDE_DIR ${PROJECT_SOURCE_DIR}/external/dlpack/include)
+include_directories(${DLPACK_INCLUDE_DIR})
```

Comment:
>include_directories [](start = 0, length = 19)

could we scope this to specific targets? looks like it's only used in dlpack_convertor.h

Reply:
good catch. 
Moved to onnxruntime_python.cmake .